el/docs/userguide/src/main/jbake/content/config.inc

```diff
@@ -0,0 +1,143 @@
+    terms of the Eclipse Public License v. 2.0, which is available at
+    http://www.eclipse.org/legal/epl-2.0.
+
+    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the
+    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+    version 2 with the GNU Classpath Exception, which is available at
+    https://www.gnu.org/software/classpath/license.html.
+
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+///////////////////////////////////////////////////////////////////////////////
+
+///////////////////////////////////////////////////////////////////////
+NOTE TO WRITERS:
+The following sections should be customized for the technology.
+This text was originally from the {TechnologyShortName} TCK.  Most references
+to {TechnologyShortName} have been parameterized to serve as a simple starting
+point for customization.  There are still many details that will
+need to be changed or removed.  The major sections 4.1, 4.2, and
+4.3 should be preserved.  If their titles are changed, the links
+at the top of config.adoc will need to be changed as well as well
+as toc.adoc.
+///////////////////////////////////////////////////////////////////////
+
+[[GBFVU]][[configuring-your-environment-to-run-the-tck-against-the-reference-implementation]]
+
+4.1 Configuring Your Environment to Run the TCK Against a Compatible Implementation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After configuring your environment as described in this section,
+continue with the instructions in link:using.html#GBFWO[Chapter 5, "Executing Tests."]
+
+
+[NOTE]
+=======================================================================
+
+In these instructions, variables in angle brackets need to be expanded
+for each platform. For example, `<JAVA_HOME>` becomes `$JAVA_HOME` on
+Solaris/Linux and `%JAVA_HOME%` on Windows. In addition, the forward
+slashes (`/`) used in all of the examples need to be replaced with
+backslashes (`\`) for Windows. Finally, be sure to use the appropriate
+separator for your operating system when specifying multiple path
+entries (`;` on Windows, `:` on UNIX/Linux).
+
+On Windows, you must escape any backslashes with an extra backslash in
+path separators used in any of the following properties, or use forward
+slashes as a path separator instead.
+
+=======================================================================
+
+
+1.  Set the following environment variables in your shell environment:
+  a.  `JAVA_HOME` to the directory in which Java SE {SEversion} is installed
+  b.  `M2_HOME` to the directory in which the Apache Maven build tool is installed.
+  c.  `PATH` to include the following directories: `JAVA_HOME/bin`,
+  and `M2_HOME/bin`
+2.  Set the following System properties:
+  a.  Set the `webServerHost` property to the name of the host on which
```

Comment:
webserverhost property is optional?

Reply:
webServerHost configuration is required to be configured